tests/foreman/cli/test_http_proxy.py

```diff
@@ -85,33 +85,55 @@ def test_positive_create_update_delete(module_org, module_location, target_sat):
 
 
 @pytest.mark.tier3
-@pytest.mark.run_in_one_thread
-@pytest.mark.stubbed
-def test_insights_client_registration_with_http_proxy():
+@pytest.mark.no_containers
+@pytest.mark.rhel_ver_match(r'^(?!6$)\d+$')
+@pytest.mark.parametrize(
+    'setup_http_proxy',
+    [True, False],
+    indirect=True,
+    ids=['auth_http_proxy', 'unauth_http_proxy'],
+)
+def test_insights_client_registration_with_http_proxy(
+    module_target_sat,
+    setup_http_proxy,
+    rhel_contenthost,
+    rhcloud_activation_key,
+    rhcloud_manifest_org,
+):
     """Verify that insights-client registration work with http proxy.
 
     :id: 5158d5c1-2b88-4c05-914b-f1c53656ffc2
 
-    :customerscenario: true
+    :parametrized: yes
 
-    :steps:
-        1. Create HTTP Proxy.
-        2. Set created proxy as "Default HTTP Proxy" in settings.
-        3. Edit /etc/resolv.conf and comment out all entries so that
-            satellite can not directly communicate outside. Ensure that
-            NetworkManger won't change it.
```

Comment:
As we're still following steps as part of setup with fixture, so can we keep this steps?

Reply:
@Gauravtalreja1 PoC for the above https://github.com/SatelliteQE/robottelo/pull/14754